client/rest-high-level/src/main/java/org/elasticsearch/client/security/DeleteUserResponse.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Licensed to Elasticsearch under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.elasticsearch.client.security;
+
+import org.elasticsearch.common.ParseField;
+import org.elasticsearch.common.xcontent.ConstructingObjectParser;
+import org.elasticsearch.common.xcontent.XContentParser;
+
+import java.io.IOException;
+
+import static org.elasticsearch.common.xcontent.ConstructingObjectParser.constructorArg;
+
+/**
+ * Response for a role being deleted from the native realm
+ */
+public final class DeleteUserResponse {
```

Comment:
this could probably extend `AcknowledgedResponse`. See `StartRollupJobResponse` for what I mean. It is just a generic `{"name": boolean}` class that allows you to specify `name`.

Reply:
I would prefer not to extend that class as it seems weird to have a dependency in this class to rollup package. In addition the response is not an acknowledge but a boolean defining if the user was found or not.

If you still thing it is better to use `AcknowledgedResponse` as a base class I will change it.

Reply:
its being moved to core in another PR thats currently in flight ;)

Reply:
I also think its worth potentially renaming it because this is really just `{"SomethingNamed": boolean}`, and it just started as acknowledged response.

Reply:
Touche :) I will implement it that way then

Reply:
I reimplement it by extending `AcknowledgedResponse`.